### PR TITLE
Fix: Resolve several PhanPluginSuspiciousParamPosition cases

### PR DESCRIPTION
**Pull request by mdeweerd**
# Fix: Resolve several PhanPluginSuspiciousParamPosition cases

Resolved several PhanPluginSuspiciousParamPosition cases.

### CODE REVIEW

**Comment by mdeweerd** [comment]
@eldy Merge conflict resolved.

File: htdocs/comm/mailing/card.php [hunk 1]
```diff
@@ -1036,7 +1036,7 @@
 				$morehtmlright .= ') &nbsp; ';
 			}
 
-			dol_banner_tab($object, 'id', $linkback, 1, 'rowid', 'ref', $morehtmlref, '', 0, '', $morehtmlright);
+			dol_banner_tab($object, 'id', $linkback, 1, 'rowid', 'ref', $morehtmlref, '', 0, '', '', 0, $morehtmlright);
```
Comment (eldy) [inline htdocs/comm/mailing/card.php:1039]: Like all other cases in the PR, here the trouble is not in the position of parameters. 
But the name of the string. It is currently named $morehtmlright when it should be $morehtmlstatus. But position and behaviour is good.

Comment (mdeweerd) [reply htdocs/comm/mailing/card.php:1039]: Ok, I will rename the string then.

Comment (mdeweerd) [reply htdocs/comm/mailing/card.php]: Renamed !

**Pull request merged**